### PR TITLE
feat: add callout displaying intake state on landing page

### PR DESCRIPTION
**Pull request by matthieu-foucault**
Implements #699 + adds second intake in dev and test data

### CODE REVIEW

File: app/pages/index.tsx [hunk 1]
```diff
@@ -25,13 +28,26 @@ const StyledBtnContainer = styled('div')`
...
+// while a `strong` element may be displayed as bold,
+// that should not be relied upon and font-weight should be used instead.
+// Using a `strong` element is still useful on a semantic level.
+const BoldText = styled('strong')`
+  font-weight: bold;
+`;
+
+const StyledCallout = styled(Callout)`
+  margintop: 16px;
+  marginbottom: 16px;
```
Comment (marcellmueller) [inline app/pages/index.tsx:40]:
```suggestion
  margin-top: 16px;
  margin-bottom: 16px;
```

Or just `margin: 16px 0` would be better. Though I don't believe this is will actually do anything anyways as it is already the default element style:

margin

**Review by marcellmueller — CHANGES_REQUESTED** [review]
Looks great other than that one issue I commented on.

File: app/pages/index.tsx (hunk 1 above)
Comment (matthieu-foucault) [reply app/pages/index.tsx:40]: Good catch! This is a half-baked refactor :sweat_smile:

Comment (marcellmueller) [reply app/pages/index.tsx:40]: I'd say so, it is cleaner and ensures the value of margin on both sides is identical. Though if we were using a global margin variable to ensure they were coming from the same constant I could see why you would think it's easier to read.

Comment (matthieu-foucault) [reply app/pages/index.tsx]: I was hesitating between multiple options. Now there's both the shorthand property, and a `spacing` section in our theme.

Comment (marcellmueller) [reply app/pages/index.tsx]: Love it, nice job.

**Review by marcellmueller — APPROVED** [review]

**Pull request merged**